boa_engine/src/tests.rs

```diff
@@ -1326,6 +1338,29 @@ fn assignment_to_non_assignable() {
     }
 }
 
+#[test]
+fn assignment_to_non_assignable_ctd() {
+    let mut context = Context::default();
+
+    let test_cases = [
+        "(()=>{})() -= 5",
+        "(()=>{})() *= 5",
+        "(()=>{})() /= 5",
+        "(()=>{})() %= 5",
+        "(()=>{})() &= 5",
+        "(()=>{})() ^= 5",
+        "(()=>{})() |= 5",
+        "(()=>{})() += 5",
+        "(()=>{})() = 5",
+    ];
+
+    for case in &test_cases {
+        let string = forward(&mut context, case);
+
+        assert!(string.starts_with("Uncaught \"SyntaxError\": "));
+    }
```

Comment:
We have a better way of testing in rust with https://github.com/boa-dev/boa/blob/main/boa_engine/src/lib.rs#L151-L193
Here is an example https://github.com/boa-dev/boa/blob/main/boa_engine/src/tests.rs#L1360-L1368

Reply:
Okay, I'll change that over; I was following the pattern I observed with similar test cases.